sites/ogj/server/templates/website-section/ogj-survey-downloads.marko

```diff
@@ -0,0 +1,171 @@
+import { get, getAsObject } from '@base-cms/object-path';
+import getAdUnit from '@endeavorb2b/base-website-common/utils/gam/get-adunit';
+import hierarchyAliases from '@endeavorb2b/base-website-common/utils/website-section/hierarchy-aliases';
+
+$ const { site } = out.global;
+$ const section = getAsObject(data, 'section');
+$ const aliases = hierarchyAliases(section);
+<theme-pennwell-website-section-layout section=section>
+  <@head>
+    <endeavor-ad-gam-head />
+  </@head>
+  <@above-container>
+    <endeavor-gam-ad-unit-out-of-page name="wa" aliases=aliases />
+    <endeavor-gam-ad-unit-out-of-page name="reskin" aliases=aliases  />
+    <endeavor-gam-ad-unit-define-display name="lb1" modifiers=["top-of-page"] />
+  </@above-container>
+
+  <div class="page-wrapper default-page">
+    <div class="row">
+      <div class="col">
+        <div class="page-wrapper__header">
+          <endeavor-breadcrumbs-website-section section=section />
+          <h1 class="page-wrapper__title">${section.name}</h1>
+          <if(section.description)>
+            <p class="page-wrapper__description">${section.description}</p>
+          </if>
+          <p class="page-wrapper__description">
+            <a href="https://ogjresearch.com" target="_blank">Buy Additional Oil & Gas Surveys</a>
+          </p>
+          <ul>
+            <li><a href="#top200">Top 200/100</a></li>
+            <li><a href="#worldwide-construction">Worldwide Construction</a></li>
+            <li><a href="#worldwide-gas-processing">Worldwide Gas Processing</a></li>
+            <li><a href="#worldwide-refining">Worldwide Refining</a></li>
+            <li><a href="#worldwide-processing">Worldwide Processing</a></li>
+            <li><a href="#ethylene-report">Ethylene Report</a></li>
+            <li><a href="#worldwide-eor">Worldwide EOR</a></li>
+            <li><a href="#catalyst-compilation">Catalyst Compilation</a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="row" id="top200">
```

Comment:
Make into a single left col and right ad rail... place section lists inside the left col